comps/dataprep/src/integrations/redis.py

```diff
@@ -620,3 +627,15 @@ async def delete_files(self, file_path: str = Body(..., embed=True)):
             if logflag:
```

Comment:
I'm not able to comment on the right line, but for `delete_files` with the file_path as "all", it's deleting from `KEY_INDEX_NAME` and `INDEX_NAME`, but it won't delete from any "custom" indices.

And for a single file delete (when it's not "all"), do you need an index_name parameter for that as well?